package.json

```diff
@@ -11,13 +11,15 @@
   },
   "files": [
     "dist/",
-    "src/index.d.ts"
+    "src/**/*.ts",
+    "src/**/*.js",
+    "src/**/*.jsx"
   ],
   "description": "Provides chat support to AmazonConnect customers",
   "scripts": {
     "test": "jest",
     "test:watch": "jest --watchAll",
-    "release": "tsc && jest && webpack --mode=production",
+    "release": "tsc --declaration && jest && webpack --mode=production",
     "devo": "jest && webpack --mode=development",
```

Comment:
Now we can not see `index.d.ts` file -- 
1st time - we need to manually compare with existing `index.d.ts` file to make sure we did not introduce backwards incompatible types. 
Our Customers typescript code might get ton of errors when they take the new version of ChatJS -- if the new auto gen typings are not backwards compatible ?